langchain4j-azure-open-ai/src/main/java/dev/langchain4j/model/azure/AzureOpenAiChatModel.java

```diff
@@ -228,28 +241,54 @@ private AzureOpenAiChatModel(String deploymentName,
         this.dataSources = dataSources;
         this.enhancements = enhancements;
         this.seed = seed;
-        this.responseFormat = responseFormat;
+        this.chatCompletionsResponseFormat = toAzureOpenAiResponseFormat(responseFormat, strictJsonSchema);
+        this.strictJsonSchema = getOrDefault(strictJsonSchema, false);
         this.listeners = listeners == null ? emptyList() : new ArrayList<>(listeners);
     }
 
+    @Override
+    public Set<Capability> supportedCapabilities() {
+        Set<Capability> capabilities = new HashSet<>();
+        if (chatCompletionsResponseFormat != null && chatCompletionsResponseFormat instanceof ChatCompletionsJsonResponseFormat) {
```

Comment:
Shouldn't it be `ChatCompletionsJsonSchemaResponseFormat`?

Reply:
That was a bug, it's fixed.